### PR TITLE
Add BBE for GraphQL subscriptions

### PR DESCRIPTION
**Pull request by Nuvindu**
## Purpose
$title.

### CODE REVIEW

File: examples/graphql-subscriptions/graphql_subscriptions.bal [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        return self.names;
+    }
+    
+    // A resource function with the `subscribe` accessor represents a field in the root `Subscription` operation.
+    // It must always return a stream. And each name will be returned in the `string` type as GraphQL responses.
+    resource function subscribe names() returns stream<string, error?> {
+        
+        return self.names.toStream();
+    
+    }
```
Comment (ThisaruGuruge) [inline examples/graphql-subscriptions/graphql_subscriptions.bal:26]:
```suggestion
        return self.names.toStream();
    }
```

File: examples/graphql-subscriptions/graphql_subscriptions.metatags [hunk 2]
```diff
@@ -0,0 +1,2 @@
+description: A GraphQL service endpoint written in Ballerina
+keywords: ballerina, ballerina by example, bbe, graphql, graphql subscription
```
Comment (ThisaruGuruge) [inline examples/graphql-subscriptions/graphql_subscriptions.metatags:2]:
```suggestion
keywords: ballerina, ballerina by example, bbe, graphql, subscription
```

File: examples/graphql-subscriptions/graphql_subscriptions.description [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// A resource function with the `subscribe` accessor inside a GraphQL service
+// represents a field in the root `Subscription` object type. Therefore, 
+// if a resource function with the `subscribe` accessor is present inside
+// the Ballerina GraphQL service, the auto-generated schema will have a
+// `Subscription` type. Each resource function with the `subscribe` accessor
+// in the service will be added as a field of the `Subscription` type.
+// The field name will be the resource function name and the field type will
+// be the constraint type of the stream returned from the resource function.
```
Comment (ThisaruGuruge) [inline examples/graphql-subscriptions/graphql_subscriptions.description:8]: Check the other place as well.

```suggestion
// A resource function with the `subscribe` accessor inside a GraphQL service
// represents a field in the root `Subscription` object type. Therefore, 
// if a resource function with the `subscribe` accessor is present inside
// the Ballerina GraphQL service, the auto-generated schema will have a
// `Subscription` type. <br/><br/>
// Each resource function with the `subscribe` accessor
// in the service will be added as a field of the `Subscription` type.
// A resource function with the `subscribe` accessor must return a `stream`.
// The field name will be the resource function name and the field type will
// be the constraint type of the stream returned from the resource function.
```

File: examples/graphql-subscriptions/graphql_subscriptions.bal [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+    function init() {
+        // Initialize the array.
+        self.names = ["Walter White", "Jesse Pinkman", "Skyler White"];
+
+    }
+
+    // The mandatory resource function with the `get` accessor represents a field in the root `Query` operation.
+    resource function get names() returns string[] {
+
```
[the remark this answers is not in the document]
Comment (Nuvindu) [reply examples/graphql-subscriptions/graphql_subscriptions.bal:16]: There are spaces like these added in other examples as well. I thought they were added for the clarity

**Review by praneesha — APPROVED** [review]

**Pull request merged**